spec/requests/uploads_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+describe 'uploads' do
+  let(:upload) { create(:upload) }
+
+  context 'logged in user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.